docs/iot-devkit/devkit_dps.md

```diff
@@ -0,0 +1,149 @@
+# Device registration with DPS
+
+Microsoft Azure provides a rich set of integrated public cloud services for all your IoT solution needs. The IoT Hub Device Provisioning Service is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention, enabling customers to provision millions of devices in a secure and scalable manner.
+
+## What you learn
+
+In this tutorial, you learn:
+- How to set up the Device Provisioning Service configuration on the device.
+- How to save Unique Device Secret on STSAFE security chip.
+- How to generate X.509 certificate.
+- How to create a device enrollment entry in the Device Provisioning Service.
+
+## What you need
+
+Finish the [Getting Started Guide](./devkit-get-started.md) to:
+
+* Have your IoT DevKit connected to Wi-Fi.
+* Prepare the development environment.
+
+An active Azure subscription. If you do not have one, you can register via one of these methods:
+
+* Activate a [free 30-day trial Microsoft Azure account](https://azure.microsoft.com/free/).
+* Claim your [Azure credit](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/) if you are an MSDN or Visual Studio subscriber.
+
+Create and link IoT Hub with Device Provisioning Service instance with [Set up auto provisioning](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision).
+
+## Open the project folder
+
+### Start VS Code
+
+* Make sure your DevKit is not connected to your PC.
```

Comment:
please refer other doc to update this section.

Reply:
Nearly every doc but get-started have this part. It is a little wired to jump from a mini solution tutorial to another. Which doc should I refer to.

Reply:
If merge of this part is needed. We can open a new pr.